cmd/daprd/options/options.go

```diff
@@ -163,6 +164,8 @@ func New(origArgs []string) (*Options, error) {
 	fs.IntVar(&opts.AppHealthProbeTimeout, "app-health-probe-timeout", int(config.AppHealthConfigDefaultProbeTimeout/time.Millisecond), "Timeout for app health probes in milliseconds")
 	fs.IntVar(&opts.AppHealthThreshold, "app-health-threshold", int(config.AppHealthConfigDefaultThreshold), "Number of consecutive failures for the app to be considered unhealthy")
 	fs.StringVar(&opts.AppChannelAddress, "app-channel-address", runtime.DefaultChannelAddress, "The network address the application listens on")
+	fs.BoolVar(&opts.EnableWorkflowVerboseLogging, "enable-wf-verbose-logging", false, "Enable verbose logging for workflows. Will increase log verbosity on workflow activity")
+	fs.MarkDeprecated("enable-wf-verbose-logging", "temporary flag for verbose logging in workflows. It could have an impact on ephemeral storage usage and potential eviction in some cases.")
```

Comment:
```suggestion
	fs.BoolVar(&opts.EnableWorkflowVerboseLogging, "enable-wf-verbose-logging", false, "Enable verbose logging of workflows. This will significantly increase the log verbosity when running workflow activities which can result in memory or disk exhaustion if logs are being collected.")
	fs.MarkDeprecated("enable-wf-verbose-logging", "This flag is temporary. Enabling this flag will significantly increase the log verbosity when running workflow activities which can result in memory or disk exhaustion if logs are being collected.")
```

Reply:
merged your commit but had to force push because of doc